src/main/groovy/de/dkfz/roddy/execution/jobs/BatchEuphoriaJobManager.groovy

```diff
@@ -27,8 +26,6 @@ import java.util.concurrent.TimeoutException
 @CompileStatic
 abstract class BatchEuphoriaJobManager<C extends Command> {
 
-    private static final LoggerWrapper logger = LoggerWrapper.getLogger(BatchEuphoriaJobManager.class.getSimpleName())
```

Comment:
Why did you remove all the loggers?

Reply:
I just removed the logger that were unused